src/Analysis/GWASV2/Components/Covariates/CustomDichotomousCovariates.stories.jsx

```diff
@@ -25,20 +25,24 @@ const studyPopulationCohort = {
   size: 400000,
 };
 
-export const SuccessCase = Template.bind({});
-SuccessCase.args = {
+export const SuccessAndZeroOverlapCases = Template.bind({});
+SuccessAndZeroOverlapCases.args = {
   dispatch: (payload) => {
     console.log('dummy dispatch', payload);
   },
-  setMode: (mode) => {
-    console.log('dummy setMode', mode);
+  handleSelect: (selectedPair) => {
+    console.log('dummy handleSelect', selectedPair);
+  },
+  handleClose: () => {
+    console.log('dummy handleClose');
   },
-  type: 'outcome',
   studyPopulationCohort: studyPopulationCohort,
   covariates: [],
   outcome: null,
+  submitButtonLabel: 'Submit!!'
```

Comment:
Do we want to have this be ```Submit``` instead of ```Submit!!``` ?

Reply:
Yes, I added that to test whether the default values would be overwritten. I can also use "Add" in this case...or "Test label for submit button"... Or maybe leave it out and let the default value render...and then test these scenarios in Jest/unit test. Any preference?

Reply:
Ah ok, I thought it might have been left in by mistake. Any of the options you suggested sound good to me.

Reply:
New ticket to follow-up on the Jest/unit tests as preferred option: https://ctds-planx.atlassian.net/browse/VADC-450